zk-tests/test.sh

```diff
@@ -9,11 +9,33 @@ SOLC="solc-${SOLC_VERSION}"
 BINARY_PATH="${REPO_ROOT}/target/release/forge"
 ERA_TEST_NODE_VERSION="v0.1.0-alpha.15"
 ERA_TEST_NODE_PID=0
+RPC_URL="http://localhost:8011"
+PRIVATE_KEY="0x3d3cbc973389cb26f657686445bcc75662b415b656078503592ac8c1abb8810e"
 
 function cleanup() {
   echo "Cleaning up..."
   stop_era_test_node
+  cleanup() {
+
+  # Force stop era test node
+  # era-test-node port
+  PORT=8011
+
+  # Using lsof to find the PID of the process using the specified port
+  PID=$(lsof -ti:$PORT)
+
+  # If a PID is found, kill the process
+  if [ ! -z "$PID" ]; then
+    echo "Killing process on port $PORT with PID $PID"
+    kill -9 $PID
```

Comment:
It could be a mac issue now that you say it. Let me try some extra things, if this is the issue I would remove this code because it will mainly run in the CI but testing manually was tedious in mac haha